typedoc.json

```diff
@@ -1,6 +1,9 @@
 {
     "entryPoints": ["./src"],
     "entryPointStrategy": "expand",
+    "compilerOptions": {
+        "outFile": null,
```

Comment:
@taylorn16 found this solution for `typedoc` which felt fairly reasonable. Any objections?

Reply:
This is just to prevent typedoc's generation step from outputting any `.ts*` files when it uses `tsc` under the hood?

Reply:
Well, `typedoc` was previously failing with the following error when I didn't have this:

```
error TS6082: Only 'amd' and 'system' modules are supported alongside --outFile.
```

I figured we don't need / want typedoc emitting anything besides the docs themselves anyway, right?